### PR TITLE
WD-12072 - docs: candid docs

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- add instructions for setting up Juju with Candid

## QA

- Follow the instructions in the doc and check that you can log in to the dashboard.

## Details

https://warthogs.atlassian.net/browse/WD-12072

### CODE REVIEW

**Review by Lukewh — APPROVED** [review]
:+1: LGTM

File: HACKING.md [hunk 1]
```diff
@@ -407,6 +408,90 @@ multipass delete juju
...
+`config.local.js`:
+
+```shell
+controllerAPIEndpoint: "wss://[container.ip]:17070/api",
+```
+
+You may also need to configure your dashboard to work with a local controller:
+
+```shell
+isJuju: true,
```
Comment (vladimir-cucu) [inline HACKING.md:490]: Maybe we could also add the fact that the following should be set as well `identityProviderAvailable: true`.

**Review by vladimir-cucu — APPROVED** [review]
LGTM! QA'ed all the steps and it works as expected. :rocket:

File: HACKING.md (hunk 1 above)
Comment (huwshimi) [reply HACKING.md:490]: No need anymore! I removed that config option in https://github.com/canonical/juju-dashboard/pull/1768.

**Pull request merged**